### DISCUSSION TITLE
Can I add Slash commands that only I can see?

### DISCUSSION THREAD

**Discussion by Wren-ofNetFae** [discord-net/Discord.Net#2828] [Q&A] [answered]
I want to have admin commands that only I can run AND only I can see.  I don't want these commands to be visible to everyone.

Is there a good way to do this?  I am struggling to find all the pieces I need to accomplish this.

My best guess seems to be to only register these admin commands in a specific test server that I own.
To that end:
Is there a way to add a `InteractionModuleBase<SocketInteractionContext>` only to a specific guild?

**Comment by Misha-133** [comment, marked as answer]
Add a `[DontAutoRegister]` attribute on that module and then register it with `InteractionService.AddModulesToGuildAsync()`
